volatility3/framework/layers/intel.py

```diff
@@ -75,7 +75,7 @@ def minimum_address(cls) -> int:
     @classproperty
     @functools.lru_cache()
     def maximum_address(cls) -> int:
-        return (1 << cls._maxvirtaddr) - 1
+        return (1 << cls._bits_per_register) - 1
```

Comment:
This is not correct, the maximum virtual address is the maximum address of the layer, not how many bits are used to create a register.  Addresses above the previous maximum value will not map properly.  This is also relied upon behaviour, changing this core constant of the intel layer would require all use of the exposed `maxium_address` value to be examined.

Reply:
What do you mean by "will not map properly" ?